facade/host.go

```diff
@@ -61,6 +61,21 @@ func (f *Facade) AddHost(ctx datastore.Context, entity *host.Host) ([]byte, erro
 	return key, alog.Error(err)
 }
 
+// AddHost registers a host with serviced. Returns the host's _public_ key.
+// Returns an error if host already exists or if the host's IP is a virtual IP.
+func (f *Facade) AddHostPrivate(ctx datastore.Context, entity *host.Host) ([]byte, error) {
+	defer ctx.Metrics().Stop(ctx.Metrics().Start("Facade.AddHostPrivate"))
+	alog := f.auditLogger.Message(ctx, "Adding Host").Action(audit.Add).Entity(entity)
```

Comment:
Make the audit log message unique "Adding Host with common key" or something like that